### ISSUE TITLE
Track number of recovered cases

### ISSUE THREAD

**Issue by Medath** [iceweasel1/COVID-19-Germany#4]
I think being able to see how many people have recovered again and therefore being able to calculate how many are still infected in a given region is an important measure to assess the current "danger" of a region.
Collecting all this data with sources might be a hard task and I don't know if it's possible to achieve on a regional level, but it would be nice

**Comment by iceweasel1** [comment]
I'll see what I can do. As you say, information policy is even harder on cures than confirmed cases.

Unfortunately, this is a major problem with our German authorities. With such a sensitive and global issue, the information policy of the authorities must actually function smoothly.

As I said, I will see if I can implement your proposal. But it may take a few days before I can present something.

**Comment by designerfuzzi** [comment]
as they (gesundeheitsämter and RKI) slooowly get the point why fast and accurate information policy can safe lifes and why this is much less stressing for a 2.5++ million people city like berlin. 
Really like that you choose to give the cases an increasing number.
Makes it easy to extend the csv..

as in the app i develop for ios i could implement polygons/arrow-connections showing which annotation in a map is originating from another annotation/case-number. Well even if it is not easy to track this down in a couple of days cause of the number of people involved then.

I'd like to link back to you from my repo that basically relies on your great supportive work.
Hope thats fine for you, let me know.

greetings

**Comment by iceweasel1** [comment]
@designerfuzzi Yeah, you said it. A fast and accurate information policy can save lives. Why our authorities are failing in this regard is not clear to me. We are living in a digital world in 2020 and the authorities (ministries in this case) in Germany are still functioning like in the stone age. No idea why. A pity. Awful...

The topic with the cases that are connected to other (German) infection cases, I have been running since the beginning of my recording in Excel. The csv-data I offer here in the repo are as given in the readme, only the most important ones, if you want to create maps or something similar with it.

The problem is that the authorities very rarely report on how the case is related to another case. In many cases there are proven connections, this is verified by the authorities, but this data is not published. Of the 218 (as of now) confirmed cases, I would have to lie about the quick, maybe 20 cases where I could note the connection to another case because it was published. An example of the federal state Baden-Württemberg in the screenshot.

Baden-Württemberg only publishes counties. Other federal states more often also the city/community. I mean for the map visualization this is enough (county). This is okay. I would like to write down cities as well. You can't identify these people anyway, if the name of the city is included. Age and gender are slightly more common in government communications than connections to other cases.

A never ending story with the red tape.

The link is of course okay. You may. I am pleased.

![bw](https://user-images.githubusercontent.com/49625256/75826283-74e04d80-5da7-11ea-8410-0e2ffa151aae.PNG)
